### ISSUE TITLE
All Plex Home users logging to one Trakt Account

### ISSUE THREAD

**Issue by rjsears** [trakt/Plex-Trakt-Scrobbler#475]
I just installed Trakt on my first Plex server (I have 4) to try it out. Seems interesting, but I have 15 home users and anytime any of those watch anything it gets send to my Trakt account.

I read about filtering just my user, but I want all users info to be collected to their own Trakt account.

I am trying to figure out how to get each of my home users set up on their own Trakt account so that I can use it to synchronize all activity for all users across all 4 plex servers. 

Maybe I am beating my head against the wall here and this is not an option...please let me know...

**Comment by rectifyer** [comment]
Have you followed this?
https://github.com/trakt-for-plex/configuration/issues/1

Basically, you need to authenticate each Trakt user in this config tool, and map them to the Plex user.

**Comment by rjsears** [comment]
Yes, and this seems to work for Friends but not for Home users.

**Comment by rectifyer** [comment]
Ah I see. Hmm, I'm not sure if that is possible yet. I have only used it with friends like you said.

**Comment by rjsears** [comment]
Yeah, I could not figure out how to do it with home users, so I am thinking it might not work that way yet!!

**Comment by dgw** [comment]
I just had one of my Home users (who is also a Friend) scrobble something to my Trakt profile earlier today. Holding on to the plugin log just in case, but usually I don't have this problem.

**Comment by dgw** [comment]
Happened a second time this morning. Strange that it should suddenly become an issue after months of working just fine…

**Comment by calibius** [comment]
Try this: [https://github.com/trakt-for-plex/configuration/issues/1#issuecomment-131524328](https://github.com/trakt-for-plex/configuration/issues/1#issuecomment-131524328)

**Comment by binarymelon** [comment]
I'm also seeing this behaviour.  It doesn't really seem intermittent like some others have suggested.  All of the shows my wife watches are getting scrobbled to my trakt account.  This is pretty disappointing.  It's the main reason I wanted Plex Pass so I could set this up.

**Comment by binarymelon** [comment]
Quick update: Whenever I manually push up the viewing activity for my wife's account it gets sent to the correct trakt account.

**Comment by antigravity83** [comment]
Yeah this plugin is completely broken for Plex servers with Home Users.

**Comment by cryptochrome** [comment]
I can confirm this issue.

**Comment by MatthewSerinity** [comment]
Also having this issue

**Comment by berot3** [comment]
So I have to create for all home users a own plex-account? :(

**Comment by calibius** [comment]
> So I have to create for all home users a own plex-account? :(

No you can use managed users . Follow the instructions in the wiki to map a managed user user account. 

http://trakt-for-plex.github.io/configuration/

https://github.com/trakt-for-plex/configuration/issues/1#issuecomment-131524328
